src/Car.java

```diff
@@ -0,0 +1,61 @@
+public class Car {
+    private String make = "Tesla";
+    private String model = "Model X";
+    private String color = "Gray";
+    private int doors = 2;
+    private boolean convertible = false;
+
+    public String getMake() {
+        return make;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public int getDoors() {
+        return doors;
+    }
+
+    public boolean isConvertible() {
+        return convertible;
+    }
+
+    public void setMake(String make) {
+       if (make == null) make = "Unknown";
+       String lowercaseMake = make.toLowerCase();
+        switch (lowercaseMake) {
+            case "holden", "porsche", "tesla" -> this.make = make;
+            default -> {
+                this.make = "Unsupported";
+            }
+
+       }
+
+    }
+
+    public void setModel(String model) {
+        this.model = model;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public void setDoors(int doors) {
+        this.doors = doors;
+    }
+
+    public void setConvertible(boolean convertible) {
+        this.convertible = convertible;
+    }
+
+    public void describeCar (){
+        System.out.println("Car " + make +" " + model + " " + color + " " + doors + " " + (convertible ? "Convertible" : "Unconvertible"));
```

Comment:
You can use more elegant solution using String.format() function:
`System.out.println(String.format("Car %s %s %s %d %s", make, model, color, doors, (convertible ? "Convertible" : "Unconvertible"));`

Reply:
You have one mistake: to less parenthesis

Reply:
Right, sorry